apps/elixir_ls_debugger/test/debugger_test.exs

```diff
@@ -166,4 +166,66 @@ defmodule ElixirLS.Debugger.ServerTest do
       assert(:hello in :int.interpreted())
     end)
   end
+
+  describe "Watch section" do
+    test "Evaluate expression with OK result", %{server: server} do
+      packet = %{
+        "arguments" => %{
+          "context" => "watch",
+          "expression" => "1 + 2 + 3 + 4",
+          "frameId" => 123
+        },
+        "command" => "evaluate",
+        "seq" => 1,
+        "type" => "request"
+      }
+
+      Server.receive_packet(
+        server,
+        packet
+      )
+
+      assert_receive(
+        %{
+          "body" => %{"result" => "10", "variablesReference" => 0},
+          "command" => "evaluate",
+          "request_seq" => 1,
+          "seq" => _,
+          "success" => true,
+          "type" => "response"
+        },
+        1000
+      )
+    end
+
+    test "Evaluate expression with ERROR result", %{server: server} do
+      packet = %{
+        "arguments" => %{
+          "context" => "watch",
+          "expression" => "1 = 2",
```

Comment:
We need to test a few more errorous expressions
`throw("abc")`
`exit("abc")`
`Process.exit(self())`

Reply:
And something that will timeout
```
receive do
:ok -> :ok
end
```

Reply:
Done with it.


![image](https://user-images.githubusercontent.com/15660128/89410227-00318680-d72c-11ea-9f33-21ecef0b725e.png)